quisp/modules/Application/Application.h

```diff
@@ -20,28 +20,27 @@ namespace modules {
  *
  *  \brief Application
  */
-class Application : public cSimpleModule {
- private:
+class Application : public IApplication {
+ public:
+  Application();
+  ~Application() {}
+
+ protected:
   int my_address;
 
-  int *other_end_node_addresses;
-  int num_of_other_end_nodes;
+  std::vector<int> other_end_node_addresses;
```

Comment:
I don't want to use a raw pointer to describe a list of addresses because I don't want to manage a memory manually.

Reply:
I agree with this

Reply:
yes! always use standard container class if possible!